main.go

```diff
@@ -21,7 +21,7 @@ func main() {
 	opts := slog.HandlerOptions{
 		Level: slog.LevelDebug,
 	}
-	handler := slog.NewJSONHandler(os.Stderr, &opts)
+	handler := slog.NewTextHandler(os.Stderr, &opts)
```

Comment:
does the text handler still handle JSONs, if not is that an issue?

Reply:
This really just pertains to the format which is output to the terminal aka `stdout`, `stderr`

## json handler
```
turk@goose:~/code/nailivic$ go run .
{"time":"2024-08-24T16:47:04.832438957-06:00","level":"DEBUG","msg":"file found","file":"hello.txt","contents":"hi there\n"}
{"time":"2024-08-24T16:47:04.832590063-06:00","level":"INFO","msg":"starting server","port":8008}
Server is running on port 8008
{"time":"2024-08-24T16:47:19.101714728-06:00","level":"DEBUG","msg":"serving response","request":"/","method":"GET","remote":"127.0.0.1:57836"}
^Csignal: interrupt
```

## text handler
```
turk@goose:~/code/nailivic$ go run .
time=2024-08-24T16:51:28.728-06:00 level=DEBUG msg="file found" file=hello.txt contents="hi there\n"
time=2024-08-24T16:51:28.728-06:00 level=INFO msg="starting server" port=8008
Server is running on port 8008
^Csignal: interrupt
```

We'll expect `systemd`/`journalctl` to handle the "structured" (message + key/value pairs) logging, and it's nicer for me to read.